holidays/utils.py

```
@@ -382,4 +414,4 @@ def list_supported_financial(include_aliases: bool = True) -> dict[str, list[str
         A dictionary where key is a market code and value is a list of supported
         subdivision codes.
     """
-    return _list_supported_entities(EntityLoader.get_financial_codes(include_aliases))
+    return _list_supported_entities(EntityLoader.get_financial_codes(include_aliases))
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)